Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/TestResultSectionView.kt

```diff
@@ -86,7 +85,7 @@ constructor(
 
                 DeviceUIState.PAIRED_POSITIVE,
                 DeviceUIState.PAIRED_POSITIVE_TELETAN,
-                DeviceUIState.PAIRED_NEGATIVE -> SpannableString(formatTestResult(uiState))
+                DeviceUIState.PAIRED_NEGATIVE -> formatTestResult(context, uiState)
```

Comment:
Also this was introduced here 
https://github.com/corona-warn-app/cwa-app-android/pull/2039/files#diff-f199e04acbe811786cab2f57926351cb55e3700f4449ef116a2dfa18a2eda979R89

Reply:
adjusted to newest 1.11 base